collect_app/src/main/java/org/odk/collect/android/activities/GoogleSheetsUploaderActivity.java

```diff
@@ -567,15 +567,13 @@ public void authRequest(Uri url, HashMap<String, String> doneSoFar) {
                 getIDOfFolderWithName(GOOGLE_DRIVE_ROOT_FOLDER, null);
                 uploadInstances(selection, selectionArgs, token);
             } catch (UserRecoverableAuthException e) {
-                Timber.e(e, "Authorization requested.. The reason uploading instances failed is due to: %s ", e.getMessage());
```

Comment:
This is not really an error state. It will happen any time a user has to authorize app access to Collect so I don't think we should log it as an error. I suppose we can log it as information for local dev?